.github/workflows/rule-validate.yml

```diff
@@ -39,8 +39,8 @@ jobs:
         uses: actions/checkout@v3
         if: github.event_name == 'issue_comment'
         with:
-          repository: ${{ steps.comment-branch.outputs.head_owner }}/${{ steps.comment-branch.outputs.head_repo }}
-          ref: ${{ steps.comment-branch.outputs.head_ref }}
+          repository: ${{ steps.comment_branch.outputs.head_owner }}/${{ steps.comment_branch.outputs.head_repo }}
```

Comment:
Another follow up from https://github.com/sublime-security/sublime-rules/pull/1198/https://github.com/sublime-security/sublime-rules/pull/1195

 Still not seeing the comment triggering behavior work as expected, and hopefully this is the last issue. I can't test this until it's merged to `main`, so no guarantees (but we have a near identical workflow for test rules, so it should be close if this doesn't fix it).